catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfRenderPlugin.java

```diff
@@ -0,0 +1,472 @@
+                            processResource.getSize() / megabyte,
+                            processResource.getName());
+                    // skip processing
+                    break;
+                }
+
+                try {
+                    final BufferedImage renderedImage = render(processResource);
+
+                    if (renderedImage != null) {
+                        // thumbnail
+                        addThumbnailToMetacard(metacard, renderedImage);
+
+                        // overview
+                        if (createOverview) {
+                            try {
+                                ProcessResource overviewImageProcessResource = createOverviewImage(
+                                        metacard.getId(),
+                                        renderedImage,
+                                        metacard,
+                                        calculateOverviewWidth(renderedImage),
+                                        calculateOverviewHeight(renderedImage));
+
+                                T overviewProcessResourceItem = createResourceItem(
+                                        processResourceItem,
+                                        overviewImageProcessResource);
+                                newProcessResourceItems.add(overviewProcessResourceItem);
+                                LOGGER.trace(
+                                        "Successfully rendered and stored {} image for NITF with id={}",
+                                        OVERVIEW,
+                                        metacard.getId());
+                            } catch (IOException e) {
+                                LOGGER.debug("Unable to generate overview image from NITF.", e);
+                            }
+                        }
+
+                        // original
+                        if (storeOriginalImage) {
+                            try {
+                                ProcessResource originalImageProcessResource = createOriginalImage(
+                                        metacard.getId(),
+                                        renderedImage,
+                                        metacard);
+
+                                T originalProcessResourceItem = createResourceItem(
+                                        processResourceItem,
+                                        originalImageProcessResource);
+                                newProcessResourceItems.add(originalProcessResourceItem);
+                                LOGGER.trace(
+                                        "Successfully rendered and stored {} image for NITF with id={}",
+                                        ORIGINAL,
+                                        metacard.getId());
+                            } catch (IOException | RuntimeException e) {
+                                LOGGER.debug("Unable to generate original image from NITF.", e);
```

Comment:
nit: For this method, is there any more information that can be added to the debug statements?  I'm thinking the metacard id or title may be useful to log at this point, if possible.